src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,75 @@
+.movie-card {
+  height: 90%;
+  width: auto;
+  padding: 0;
+  margin: 20px;
+  border-radius: 5px;
+  position: relative;
+  padding-left: 10px;
```

Comment:
```suggestion
  
```

Reply:
I think this was throwing off the layout and causing some funny movements when hovered.